brut.apktool/apktool-lib/src/test/resources/aapt2/testapp/res/values/strings.xml

```diff
@@ -3,4 +3,5 @@
     <string name="app_name">testapp</string>
     <string name="long_string_32767" />
     <string name="surrogate_issue_2299">🔆</string>
+    <string name="test_string43"><ul> <li><b>aaaaa aa aaaaa</b> – aaaaaaa aaaaaaaaaa aa aaaaaaaa aaaaaa aaaaa (aaaa) aaaa aaaaaaaaa aaaaa aa aaaaaaaaa aaaaaaa aaaa</li> <li><b>aaaaaaaaa aaaaaaaaaaaaaaa aaaaaaaa</b> – aaaaaaa aaaaaaaaaa aaaaaaaaa aaaa aaaaaa aa aaaa aaaa aaaa aaaa aaaaaaa aaaaaaaaaaaaa, aaaaaa aaa aaaaaaaaaa (aaa) aaaaaaaaaaaaaaa</li> <li><b>aaaaaaaaaaa aaaaaa aaaaaaaaaa</b> – aaaaaaaaaa aaaa aaaaaa aa a aaa aa aaaaaa aaa aaaaaaa (aaaaa aaaaaaaa) aaaaaaaa aa aaaaaa aaaaa aaa aaaa</li> </ul></string>
```

Comment:
in aapt2, I tend to name the strings after the test/issue. aapt1 is messy and legacy, so I just count up as you did with test_stringXX

Reply:
yeah, should've seen that. did you change it?

Reply:
I did yes post merge - https://github.com/iBotPeaches/Apktool/commit/99166e3609ecbe997ba179cd112665ffa6fc1c73